### ISSUE TITLE
Robust support of validating multiple values

### ISSUE THREAD

**Issue by sunnyosun** [laminlabs/lamindb#2101]
### Description of feature

For instance: `[["T cell", "B cell"], "T cell", "B cell"]`

**Comment by Zethson** [comment]
Yes! Also very useful for perturbation curation. Commonly also ["PERT1+PERT2"] and other designs.

**Comment by falexwolf** [comment]
This has long been added.
